### ISSUE TITLE
Accessibility: jupyter.org

### ISSUE THREAD

**Issue by Divine-peace** [jupyterhub/outreachy#104]
### What page is this for?

https://jupyter.org/

### WAVE accessibility report

![jupyter-org](https://user-images.githubusercontent.com/103381382/197046716-0cc65781-47fb-49e0-8249-cdd87e49fcd8.png)

**Suggestions for Contrast Errors:**

Adequate contrast of text is necessary for all users, especially users with low vision.

- Increase the contrast between the foreground (text) color and the background color. Large text (larger than 18 point or 14 point bold) does not require as much contrast as smaller text.
- Change the color of the text and background.
- The contrast ratio between text and its background should be at least 4.5:1. For large-scale text (at least 18-point normal or 14-point bold), a contrast ratio of at least 3:1 between text and its background is recommended.

**Suggestions for Alert (Nearby Images have the same Alt text ):**

When two images have the same alternative text, this often causes redundancy or indicates incorrect alternative text.

- Ensure that the alternative text for each image or image button is appropriate while removing unnecessary redundancy. If the content of the image is already conveyed elsewhere (through text or the alternative text of a nearby image) or if the image does not convey content, the image may generally be given empty/null alternative text (alt="").
- Change or replace alt text for one the images.
**Suggestions for Alert (Missing first level heading heading):**
Headings facilitate page navigation for the users of many assertive technologies. They also provide semantic and visual meaning and structure to the document. 
* A first level heading `(<h1>)` should be present on nearly all pages. 
* The main heading of the page, that is, the logo should be placed within an `<h1>` element.

**Suggestions for Alert ( Possible heading):**

* From the above code evaluation, I recommend that a heading element should be used to replace the paragraph <p> element.

**Suggestions for Features (Linked image with alt text):**
* Ensure that the alternative text presents the content of the image and/or the function of the link. If the full content and function of the link is presented in text within the link (an image and a text caption both within the same link, for example), then the image should generally be given empty/null alternative text (alt="") to avoid redundancy.

**Suggestions for Features (Alternative text):**

* Ensure that the alternative text conveys the content and function of the image accurately and succinctly.
* The alt attribute should be equivalent, accurate, and succinct.

**Suggestions for Structural Elements (Heading levels):**
* Ensure that the text in question is truly a heading and that it is structured correctly in the page outline.

**Suggestions for Structural Elements(Unordered list):**
* Ensure that an unordered (bulleted) list is appropriate for the context.
 If list items are sequential or numbered, an ordered list `(<ol>)` is likely more appropriate.

**Suggestions for Structural Elements (Unordered list):**

* Ensure the element defines page navigation.
Multiple navigation elements on one page can be differentiated with ARIA labels.

**Suggestions for Structural Elements (Footer):**

* Ensure the element surrounds and defines page or page section footer content.

**Suggestions for Features (Language):**
* Ensure the language is properly identified for the page (e.g., <html lang="en">). If content within the page is in a language different from the page's language, identify it using a valid lang attribute value (e.g., <p lang="fr">

### The HTML element to be changed

```html
HTML for ( Contrast Errors):

<a class="orange-button " href="/try">
Try it in your browser
</a>

<a class="orange-button install-button" href="/install">
Install JupyterLab
</a>

<a class="orange-button install-button" href="/install">
Install the Notebook
</a>

<a class="orange-button" href="/hub">
Learn more about JupyterHub
</a>

<a class="orange-button install-button" href="/install">
Install Voilà
</a>

HTML for Alerts:
<span>
A nearby image has the same alternative text
<img class="img-responsive fade-in two" src="/assets/homepage/circle2.svg" width="677" height="286" alt="circle of programming language icons" loading="eager">
</span>

<span>
A nearby image has the same alternative text
<img class="img-responsive fade-in three" src="/assets/homepage/circle3.svg" width="677" height="286" alt="circle of programming language icons" loading="eager">
</span>

Missing first level heading
<nav class="navbar">
<div class="navbar-header">
<a class="navbar-brand" href="/">
<picture>
<source media="(max-width: 767px)" srcset="/assets/logos/logomark-orangebody-greyplanets.svg">
<source media="(min-width: 768px)" srcset="/assets/logos/rectanglelogo-greytext-orangebody-greymoons.svg">
Linked image with alternative text
<img class="navbar-logo" src="/assets/logos/rectanglelogo-greytext-orangebody-greymoons.svg" alt="Jupyter Home" height="40" loading="eager">
</picture>
</a>
</div>

Possible heading
<p class="dropshadow">
Cité des Science, Paris, France
</p>
<p class="dropshadow">
CFP open Oct 31st
</p>
```

### Your proposed HTML (or CSS), after change

```html
CSS for (7 Contrast Errors):

.orange-button {
Color: #FFFFFF;
Background-color:#903F13;
}

HTML for  Alerts (Nearby image have the same alternative text):

<img class="img-responsive fade-in three" src="/assets/homepage/circle3.svg" width="677" height="286" alt="a programming language icon" loading="eager">

HTML for  Alerts (Missing First Level Heading):
<nav class="navbar">
<div class="navbar-header">
<h1>
<a class="navbar-brand" href="/">
<picture>
<source media="(max-width: 767px)" srcset="/assets/logos/logomark-orangebody-greyplanets.svg">
<source media="(min-width: 768px)" srcset="/assets/logos/rectanglelogo-greytext-orangebody-greymoons.svg">
Linked image with alternative text
<img class="navbar-logo" src="/assets/logos/rectanglelogo-greytext-orangebody-greymoons.svg" alt="Jupyter Home" height="40" loading="eager">
</picture>
</a>
</h1>
</div>

HTML for  Alerts (Possible heading):

<h2 class="dropshadow">
Cité des Science, Paris, France
</h2>
<h2 class="dropshadow">
CFP open Oct 31st
</h2>
```

**Comment by minrk** [comment]
Thank you for the suggestions!

**Comment by Divine-peace** [comment]
Appreciated sir

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
